medplum/medplum
# Add "Save" behavior to foomedical profile

## Description

This PR resolves #4756.

## Steps to test

- [ ] Run the foomedical example
   ```bash
   examples/foomedical > npm install
   examples/foomedical > npm run dev  
    ```
- [ ] Go to the profile page (http://localhost:3000/account/profile)
- [ ] All fields should be editable, except for the email
  - `NOTE`: I disabled the email field because I wanted to confirm whether or not we will allow people to edit it in the example.
- [ ] To confirm the update was successful, visit https://app.medplum.com/ or check the network tab.

[Screencast from 02-07-2024 09:38:35.webm](https://github.com/medplum/medplum/assets/22326737/418667ac-16ff-45a5-a76c-ff54f6460f7f)
